recipes/pubgeo-core3d-metrics/pubgeo-core3d-metrics.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/core3dmetrics/geometrics/plot.py b/core3dmetrics/geometrics/plot.py
+index 0d6be25..8b01d15 100644
+--- a/core3dmetrics/geometrics/plot.py
++++ b/core3dmetrics/geometrics/plot.py
+@@ -3,9 +3,6 @@ import platform
+ import numpy as np
+ 
+ import matplotlib  as mpl
+-if os.getenv('DISPLAY') is None and not platform.system() == "Windows":
+-    import matplotlib
+-    matplotlib.use('Agg')
```

Comment:
You haven't seen any errors complaining that matplotlib has not been set to use 'Agg' when running the metrics code after your patch?  In some other scripts, I've had to unconditionally call "matplotlib.use('Agg')" or some other packages complain.  But if this works then that's fine.

Reply:
No, I have not. It may be because the template has showPlots = false so it does not try to show a window on the screen. But I also run this in the regular OS not inside a container. I will just rebuild this with the change you suggested.